homework/style.css

```diff
@@ -1,3 +1,81 @@
 .alert-error {
```

Comment:
Your app is not responsive:

![image](https://user-images.githubusercontent.com/2788771/59549802-2bbf4380-8f63-11e9-8fa4-e0bacdc8ea7e.png)